src/integrationTest/resources/application.properties

```diff
@@ -1,25 +1,25 @@
-storage.account-name: bulkscan
-storage.account-key: testkey
+storage.account-name=bulkscan
+storage.account-key=testkey
 
-service.storage-config[0].name: bulkscan
-service.storage-config[0].sas-validity: 300
-service.storage-config[0].target-storage-account: bulkscan
+service.storage-config[0].name=bulkscan
+service.storage-config[0].sas-validity=300
+service.storage-config[0].target-storage-account=bulkscan
 
-clients.error-notifications.password: password
-clients.error-notifications.url: false
-clients.error-notifications.username: username
+clients.error-notifications.password=password
+clients.error-notifications.url=false
+clients.error-notifications.username=username
 
-flyway.skip-migrations: false
+flyway.skip-migrations=false
 
 spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDriver
 spring.datasource.url=jdbc:tc:postgresql:9.6://localhost/blob_router
 
 storage-blob-processing-delay-in-minutes=0
-public-key-der-file: nonprod_public_key.der
+storage-public-key-der-file=signing/test_public_key.der
```

Comment:
public key is not related to storage in any way ?

Reply:
removed "storage" prefix